src/pages/developers/local-environment.tsx

```diff
@@ -0,0 +1,314 @@
+      image: EthDiamondBlackImage,
+    },
+    {
+      id: "epirus",
+      url: "https://www.web3labs.com/epirus",
+      githubUrl: "https://github.com/web3labs/epirus-free",
+      background: "#ffffff",
+      name: "Epirus",
+      description: "page-developers-local-environment:page-local-environment-epirus-desc",
+      alt: "page-developers-local-environment:page-local-environment-epirus-logo-alt",
+      image: EpirusImage,
+    },
+    {
+      id: "createethapp",
+      url: "https://github.com/PaulRBerg/create-eth-app",
+      githubUrl: "https://github.com/PaulRBerg/create-eth-app",
+      background: "#ffffff",
+      name: "Create Eth App",
+      description: "page-developers-local-environment:page-local-environment-eth-app-desc",
+      alt: "page-developers-local-environment:page-local-environment-eth-app-logo-alt",
+      image: EthDiamondBlackImage,
+    },
+    {
+      id: "scaffoldeth",
+      url: "https://github.com/austintgriffith/scaffold-eth",
+      githubUrl: "https://github.com/austintgriffith/scaffold-eth",
+      background: "#ffffff",
+      name: "scaffold-eth",
+      description: "page-developers-local-environment:page-local-environment-scaffold-eth-desc",
+      alt: "page-developers-local-environment:page-local-environment-scaffold-eth-logo-alt",
+      image: ScaffoldEthImage,
+    },
+    {
+      id: "soliditytemplate",
+      url: "https://github.com/paulrberg/solidity-template",
+      githubUrl: "https://github.com/paulrberg/solidity-template",
+      background: "#ffffff",
+      name: "Solidity template",
+      description: "page-developers-local-environment:page-local-environment-solidity-template-desc",
+      alt: "page-developers-local-environment:page-local-environment-solidity-template-logo-alt",
+      image: EthDiamondBlackImage,
+    },
+    {
+      id: "foundry",
+      url: "https://getfoundry.sh/",
+      githubUrl: "https://github.com/foundry-rs/foundry",
+      background: "#ffffff",
+      name: "Foundry",
+      description: "page-developers-local-environment:page-local-environment-foundry-desc",
+      alt: "page-developers-local-environment:page-local-environment-foundry-logo-alt",
+      image: FoundryImage,
+    },
+  ]
+  
+  frameworksList = await Promise.all(frameworksList.map(async (framework) => {
+    const repoData = await ghRepoData(framework.githubUrl)
+    return {
+      ...framework,
+      starCount: repoData.starCount,
+      languages: repoData.languages.slice(0,2),
+    }
+  }))
+
+  return {
+    props: {
+      ...(await serverSideTranslations(locale!, requiredNamespaces)),
+      frameworksList,
+    },
+  }
+}
+
+const LocalEnvironmentPage = ({ frameworksList }) => {
+  const { t } = useTranslation("page-developers-local-environment")
+
+  return (
+    <Flex direction="column" alignItems="center" w="full" mx="auto" mt={16}>
+      <PageMetadata
+        title={t("page-developers-local-environment:page-local-environment-setup-meta-title")}
```

Comment:
tip: not 100% sure but I think that when you need to use the first namespace declared in the `useTranslation`, you don't need to specify the namespace in the `t` function.

https://react.i18next.com/latest/usetranslation-hook#loading-namespaces
```
const { t, i18n } = useTranslation(['ns1', 'ns2', 'ns3']);
t('key'); // will be looked up from namespace ns1
```